spec/features/expense_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Expense page', type: :system do
+  describe 'Group/show page' do
+    before(:each) do
+      @user = User.create!(name: 'name', email: 'email@email.com',
+                           password: 'password', password_confirmation: 'password')
+      @group = Group.create(name: 'travel', icon: 'https://icons.com/icon1', author_id: @user.id)
+      @user.skip_confirmation!
+      @user.save!
+      visit new_user_session_path
+      fill_in 'user_email', with: @user.email
+      fill_in 'user_password', with: @user.password
+      click_button 'Sign in'
+      visit category_path(id: @group.id)
+    end
+
+    it 'displays the name of group' do
+      expect(page).to have_content(@group.name)
+    end
+
+    it 'displays a button to add a new expense' do
+      expect(page).to have_content('New expense')
+    end
+
+    it 'the new expense button redirects to a page to add new expense' do
+      click_button 'New expense'
+      expect(page).to have_current_path new_category_expense_path(category_id: @group.id)
+    end
```

Comment:
While you did a great job working on the project requirements 11 of your test cases out 39 examples are failing.  And I think this error is bing caused by the confirmations It would be nice if you fixed this issue.
![Screenshot 2023-03-02 at 7 08 12 AM](https://user-images.githubusercontent.com/79506105/222337787-c620cc14-e037-4f38-80e1-064e435fcac7.png)
![Screenshot 2023-03-02 at 7 10 05 AM](https://user-images.githubusercontent.com/79506105/222337838-50c80143-c2f1-4975-a9a1-79fb6afcccdc.png)
![Screenshot 2023-03-02 at 7 10 15 AM](https://user-images.githubusercontent.com/79506105/222338132-f0bbd220-be09-42a5-b9a6-75934e086c0b.png)

Reply:
@KayLemba Thank you for the review, the tests were because I changed te devise functionality, it is solved now